rviz_common/include/rviz_common/visualizer_app.hpp

```diff
@@ -70,6 +74,7 @@ private Q_SLOTS:
   QTimer * continue_timer_;
   VisualizationFrame * frame_;
   std::string node_name_;
+  std::unique_ptr<rviz_common::ros_integration::RosClientAbstractionIface> ros_client_;
```

Comment:
nitpick: this name `ros_client_` leaves out "abstraction", but in the other file you name the variable `node_abstraction` leaving our "ros". I don't mind leaving out ros or abstraction (though my default preference would be to include both in both variable names), but it would be best to keep them at least consistent, e.g. `client_abstraction_` or `ros_node_`.